### PR TITLE
Instructions on adding external source repositories to the application

### PR DESCRIPTION
**Pull request by ConradWeiser**
Amendment to the Guides

### CODE REVIEW

**Comment by ConradWeiser** [comment]
image
Rendered page preview

**Comment by ConradWeiser** [comment]
image

Rendered page preview

File: src/help/guides/adding-repos.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+:::
+
+
+## Adding an External Repository
+1. Open up Paperback on your iOS device
+1. Navigate to the Settings tab of the application
+1. Navigate to the Sources tab inside of Settings
+1. At the bottom of the page, you are able to add a repository by specifying a name, and the URL which the repository sits at. Ensure that your repository title is at least 4 characters long.
+    > Reference the [Known Repositories](/help/guides/adding-repos/#known-repositories) section for a list of known repositories, their links, and what they contain
+1. Tap 'Add'
```
Comment (FramboisePi) [inline src/help/guides/adding-repos.md:21]: Maybe using `"` would be more consistent with getting-started guide: `"Install AltStore"`
```suggestion
1. Tap "Add"
```

File: src/help/guides/adding-repos.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+::: warning Warning
+Currently external sources are only available on the Beta version of the application. If you are not a Patreon, this guide will not be very useful to you. This feature will be available in public builds in the future.
+:::
+
+
+## Adding an External Repository
+1. Open up Paperback on your iOS device
+1. Navigate to the Settings tab of the application
+1. Navigate to the Sources tab inside of Settings
```
[the remark this answers is not in the document]
Comment (ConradWeiser) [reply src/help/guides/adding-repos.md:18]: For consistency, that's a good idea

**Comment by ConradWeiser** [comment]
Should I add another column to the table with a list of all of the sources available in that repository? What do you think?

**Comment by Joery** [comment]
> Should I add another column to the table with a list of all of the sources available in that repository? What do you think?

Yeah I think that would look good. Maybe also include the main repository included with Paperback? So there's an easy place to guide users to in case they ask what sites are available in Paperback.

**Comment by ConradWeiser** [comment]
Page has been adjusted
image

File: src/help/guides/adding-repos.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Adding Repositories
+lang: en-US
+---
+
+# External Source Repositories
+## What is an External Repository?
+**Paperback** out of the box only comes with **MangaDex** as a source for manga. The app however has support for adding manga from external repositories which are not on the official listings. Beta Sources, 18+ Sources, and sources developed independently are seperated due to **Paperback** having to adhere to Apple policies in order to be distributed through TestFlight and the Apple App Store.
```
Comment (FramboisePi) [inline src/help/guides/adding-repos.md:8]: I personally find this section a bit confuse.

We say `**Paperback** out of the box only comes with **MangaDex** as a source for manga` in the `What is an External Repository?` Moreover I believe the Official Repository is also built into Paperback. I'm not sure about the installation process but here is what I did understood: Paperback come with MangaDex and users can add sources from the official repo. Then there is also a support for external repo.

It is also possible to add repo that are on the official listing like H-extensions.

---

Shouldn't this be `source`?
```suggestion
**Paperback** out of the box only comes with **MangaDex** as a source for manga. The app however has support for adding sources from external repositories which are not on the official listings. Beta Sources, 18+ Sources, and sources developed independently are seperated due to **Paperback** having to adhere to Apple policies in order to be distributed through TestFlight and the Apple App Store. 
```

---

I believe the issue in the section is the confusion of `the mangas from a source`, the `source` and the `source repo`.
An idea could be to explain first that other sources are supported and then that external repo can be added to add more sources.

File: src/help/guides/adding-repos.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+:::
+
+
+## Adding an External Repository
+1. Open up Paperback on your device.
+1. Navigate to the Settings tab of the application.
+1. Navigate to the Sources tab inside of Settings.
+1. At the bottom of the page, you are able to add a repository by specifying a name, and the URL which the repository sits at. Ensure that your repository title is at least 4 characters long.
+    > Reference the [Known Repositories](/help/guides/adding-repos/#known-repositories) section for a list of known repositories, their links, and what they contain.
+1. Tap "Add".
```
Comment (FramboisePi) [inline src/help/guides/adding-repos.md:21]: Could using some bold statement emphasize the interface labels? That had been used in the old getting-started guide.
```suggestion
1. Navigate to the **Settings** tab of the application.
1. Navigate to the **Sources** tab inside of Settings.
1. At the bottom of the page, you are able to add a repository by specifying a name, and the URL which the repository sits at. Ensure that your repository title is at least 4 characters long.
    > Reference the [Known Repositories](/help/guides/adding-repos/#known-repositories) section for a list of known repositories, their links, and what they contain.
1. Tap "**Add**".
```

File: src/.vuepress/config/nav/en.js [hunk 5]
```diff
@@ -11,6 +11,10 @@ module.exports = [
 						text: "Getting started",
 						link: "/help/guides/getting-started/",
 					},
+					{
+						text: "Adding External Repositories",
+						link: "/help/guides/adding-repos/"
+					}
```
Comment (FramboisePi) [inline src/.vuepress/config/nav/en.js:17]: Probably not needed but there were some coma before.
```suggestion
						link: "/help/guides/adding-repos/",
					},
```

**Pull request merged**